eyelid-match-ops/Cargo.toml

```diff
@@ -31,6 +31,7 @@ num-bigint.workspace = true
 num-traits.workspace = true
 
 bitvec.workspace = true
+colored.workspace = true
```

Comment:
Nitpick: This is a test-only dependency, so it should go in `[dev-dependencies]`.

Sometimes this is relevant for performance, because it increases the binary size.

Reply:
I can make this change.

Reply:
Simple one, thanks

Reply:
done

Reply:
done